components/icon/Icon.js

```diff
@@ -13,7 +13,7 @@ import PropTypes from 'prop-types';
 const Icon = ({ name, className, viewBox, alt, ...rest }) => {
     return (
         <>
-            <svg viewBox={viewBox} className={className} role="img" focusable="false" {...rest}>
+            <svg viewBox={viewBox} className={`icon-16p `.concat(className)} role="img" focusable="false" {...rest}>
```

Comment:
`icon-16p` was put inside `className` as default value to make it overridable, example:

```jsx
<Icon className="icon-32p fill-global-grey mr1" />
```

Reply:
yep but the size should be an arg as the name imho. Extend classnames is more common than changing the size

Reply:
both color and size are config, we should change them via attribute as we do for the name, I will add this